service/frontend/adminHandler.go

```diff
@@ -672,6 +672,42 @@ func (adh *AdminHandler) DescribeCluster(_ context.Context, _ *adminservice.Desc
 	}, nil
 }
 
+func (adh *AdminHandler) GetClusterMembers(
+	ctx context.Context,
+	request *adminservice.GetClusterMembersRequest,
+) (_ *adminservice.GetClusterMembersResponse, retError error) {
+	defer log.CapturePanic(adh.GetLogger(), &retError)
+
+	scope, sw := adh.startRequestProfile(metrics.AdminGetClusterMembersScope)
+	defer sw.Stop()
+
+	if request == nil {
+		return nil, adh.error(errRequestNotSet, scope)
+	}
+
+	metadataMgr := adh.GetClusterMetadataManager()
+
+	resp, err := metadataMgr.GetClusterMembers(&persistence.GetClusterMembersRequest{NextPageToken: nil})
```

Comment:
Do we plan to wire the input request and the persistence request?

Reply:
good catch!

Reply:
what is the default page size? do we need to be worried about more clusters than page size?

Reply:
@yiminc yes we need to take this into account, luckily there is already pagination supported. Just need to wire the nextPageToken to the persistence request

Reply:
addressed